code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -205,86 +205,86 @@
+			/obj/item/weapon/powerfist = -1,
+			/obj/item/weapon/throwing_knife = -1,
 			/obj/item/ammo_magazine/standard_smartmachinegun = 4,
 		),
 		"Grenades" = list(
-			/obj/item/explosive/grenade/frag = 600,
+			/obj/item/explosive/grenade/frag = -1,
 			/obj/item/explosive/grenade/frag/m15 = 50,
 			/obj/item/explosive/grenade/incendiary = 50,
 			/obj/item/explosive/grenade/cloakbomb = 50,
 			/obj/item/explosive/grenade/drainbomb = 10,
-			/obj/item/explosive/grenade/mirage = 100,
-			/obj/item/storage/box/m94 = 200,
+			/obj/item/explosive/grenade/mirage = -1,
+			/obj/item/storage/box/m94 = -1,
 			/obj/item/storage/box/m94/cas = 30,
 		),
 		"Attachments" = list(
-			/obj/item/attachable/bayonet = 1000,
-			/obj/item/attachable/compensator = 1000,
-			/obj/item/attachable/extended_barrel = 1000,
-			/obj/item/attachable/suppressor = 1000,
-			/obj/item/attachable/lace = 1000,
-			/obj/item/attachable/flashlight = 1000,
-			/obj/item/attachable/magnetic_harness = 1000,
-			/obj/item/attachable/reddot = 1000,
-			/obj/item/attachable/angledgrip = 1000,
-			/obj/item/attachable/bipod = 1000,
-			/obj/item/attachable/burstfire_assembly = 1000,
-			/obj/item/attachable/gyro = 1000,
-			/obj/item/attachable/lasersight = 1000,
-			/obj/item/attachable/verticalgrip = 1000,
-			/obj/item/attachable/stock/t19stock = 1000,
-			/obj/item/attachable/stock/t35stock = 1000,
-			/obj/item/attachable/attached_gun/flamer = 1000,
-			/obj/item/attachable/attached_gun/shotgun = 1000,
-			/obj/item/attachable/attached_gun/grenade = 1000,
+			/obj/item/attachable/bayonet = -1,
+			/obj/item/attachable/compensator = -1,
+			/obj/item/attachable/extended_barrel = -1,
+			/obj/item/attachable/suppressor = -1,
+			/obj/item/attachable/lace = -1,
+			/obj/item/attachable/flashlight = -1,
+			/obj/item/attachable/magnetic_harness = -1,
+			/obj/item/attachable/reddot = -1,
+			/obj/item/attachable/angledgrip = -1,
+			/obj/item/attachable/bipod = -1,
+			/obj/item/attachable/burstfire_assembly = -1,
+			/obj/item/attachable/gyro = -1,
+			/obj/item/attachable/lasersight = -1,
+			/obj/item/attachable/verticalgrip = -1,
+			/obj/item/attachable/stock/t19stock = -1,
+			/obj/item/attachable/stock/t35stock = -1,
+			/obj/item/attachable/attached_gun/flamer = -1,
+			/obj/item/attachable/attached_gun/shotgun = -1,
+			/obj/item/attachable/attached_gun/grenade = -1,
 		),
 		"Boxes" = list(
-		/obj/item/ammo_magazine/box9mm = 100,
-		/obj/item/ammo_magazine/acp = 100,
-		/obj/item/ammo_magazine/magnum = 100,
-		/obj/item/ammo_magazine/box10x24mm = 100,
-		/obj/item/ammo_magazine/box10x26mm = 100,
-		/obj/item/ammo_magazine/box10x27mm = 100,
+		/obj/item/ammo_magazine/box9mm = -1,
+		/obj/item/ammo_magazine/acp = -1,
+		/obj/item/ammo_magazine/magnum = -1,
+		/obj/item/ammo_magazine/box10x24mm = -1,
+		/obj/item/ammo_magazine/box10x26mm = -1,
+		/obj/item/ammo_magazine/box10x27mm = -1,
 		/obj/item/storage/box/visual/magazine = 30,
```

Comment:
no

Reply:
100 is virtually infinite when we are talking about ammo boxes... Especially since the ammo itself is made infinite